config/routes.rb

```diff
@@ -1,8 +1,9 @@
 Rails.application.routes.draw do
-  get "up" => "rails/health#show",  as: :rails_health_check
-  get "/api/v1/posters",            to: "api/v1/posters#index"
-  get "/api/v1/posters/:id",        to: "api/v1/posters#show"
-  post "/api/v1/posters",           to: "api/v1/posters#create"
-  patch "/api/v1/posters/:id",      to: "api/v1/posters#update"
-  delete "/api/v1/posters/:id",     to: "api/v1/posters#destroy"
-end
+  get "up" => "rails/health#show", as: :rails_health_check
+  get "/api/v1/posters",           to: "api/v1/posters#index"
+  get "/api/v1/posters/:id",       to: "api/v1/posters#show"
+  post "/api/v1/posters",          to: "api/v1/posters#create"
+  patch "/api/v1/posters/:id",     to: "api/v1/posters#update"
+  delete "/api/v1/posters/:id",    to: "api/v1/posters#destroy"
+  get "/api/v1/posters",           to: "api/v1/posters#sort"
```

Comment:
Hey Jonathan, my understanding is that we cannot have two routes to the same path pointing to two different controller actions.
Line 3's path: "get "/api/v1/posters" is pointed toward controller action #Index
Line 8's path: "get "/api/v1/posters" is pointed toward controller action #sort

After looking at the controller file, I don't see a sort action was created?
Maybe we can talk about this during our work time today?